.travis.yml

```diff
@@ -1,13 +1,16 @@
 sudo: false
 language: go
 go:
+  - "1.11.x"
   - "1.10.x"
   - "1.9.x"
-  - "1.8.x"
-  - master
+  - tip
```

Comment:
@marwan-at-work Can you please help me understand why "master" was replaced with "tip"? I tried reading the PR description and commit messages, and didn't find a rationale there.

The latest [Travis CI documentation](https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use) says:

> or use `master` to get the latest version from source

That's why this change is unexpected for me and I'd like to understand it better.

Reply:
@dmitshur im not sure but it failed for me and switched to tip because we use `tip` at athens. I don't know the difference, but maybe we can open a pr and check?

Reply:
I can do a quick experiment.

Reply:
See #1002 for successful run with `master`.